src/java.base/share/classes/java/util/Currency.java

```diff
@@ -1174,15 +1176,12 @@ private static boolean containsInconsistentInstances(
                     || prop.fraction != fractionDigit);
         }
 
-        private static boolean isPastCutoverDate(String s)
-                throws ParseException {
-            SimpleDateFormat format = new SimpleDateFormat(
-                    "yyyy-MM-dd'T'HH:mm:ss", Locale.ROOT);
-            format.setTimeZone(TimeZone.getTimeZone("UTC"));
-            format.setLenient(false);
-            long time = format.parse(s.trim()).getTime();
-            return System.currentTimeMillis() > time;
-
+        // cutOver adheres to: "yyyy-MM-ddTHH:mm:ss"
```

Comment:
In java.time, `YEAR` is represented as `u`, so "uuuu" is more accurate.

Reply:
> Looks good. Since this code does not run in normal situations (only works when users provide a custom currency properties file), further (not concise) performance refactor may not be significant.

Agreed, this change is easy and more readable IMO, so I figured might as well, but won't do anything past this.

> In java.time, YEAR is represented as u, so "uuuu" is more accurate.

I used "`yyyy-MM-ddTHH:mm:ss`" because this is the literal date/time format specified by `Currency`. (It still uses DateFormat patterns).

_The format of the timestamp must be of ISO 8601 format : 'yyyy-MM-dd'T'HH:mm:ss'_

But I understand the confusion. Changed the comment to use a generalized ISO 8601 format wording.